projects/driver_controls/inc/center_console.h

```diff
@@ -7,24 +7,50 @@
 //
 // Raises INPUT_EVENT_CENTER_CONSOLE_* events with empty data fields.
 // Assumes all inputs other than hazards are non-latching.
+//
```

Comment:
Maybe I'm missing something, but why do we need all these center console changes if you're not even compiling it in?

Reply:
Oh I didn't realize we were disabling it. What was the reasoning behind disabling this again?